### PR TITLE
Terri- Interface Change

### PR DESCRIPTION
**Pull request by TeresaGobble**

### CODE REVIEW

File: client/CrimeMap.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  latitude: number;
+  longitude: number;
+  primaryType: string;
+  description: string;
+}
+
+const CrimeMap = () => {
+  //array containing crimes that fit the user's entered criteria
+
+  console.log("The crimes Data", crimes.features[0].properties.latitude)
```
Comment (clwilshire) [inline client/CrimeMap.tsx:38]: 🤔

**Review by clwilshire — COMMENTED** [review]
Looks good!

**Review by yellowstrings — APPROVED** [review]
looks gucci!

File: client/CrimeMap.tsx (hunk 1 above)
Comment (yellowstrings) [reply client/CrimeMap.tsx:38]: WHAT is this

Comment (TeresaGobble) [reply client/CrimeMap.tsx:38]: It is a feature 💦

**Pull request merged**